.github/workflows/deploy.yaml

```diff
@@ -190,7 +190,8 @@ jobs:
                 id: builddeploy
                 uses: Azure/static-web-apps-deploy@v1
                 env:
-                    ZTOR_URL: https://${{ needs.environment-variables.outputs.ZTOR_CONTAINER_APP_NAME }}.zero.zenmo.com
+                    VITE_ZTOR_URL: https://${{ needs.environment-variables.outputs.ZTOR_CONTAINER_APP_NAME }}.zero.zenmo.com
+                    VITE_KLEINVERBRUIK_URL: http://${{ needs.environment-variables.outputs.ZTOR_CONTAINER_APP_NAME }}
```

Comment:
`VITE_KLEINVERBRUIK_URL` is not this, please remove so it falls back to the default.